src/openfluid/ui/waresdev/WaresSrcImportDialog.cpp

```diff
@@ -311,23 +323,21 @@ void WaresSrcImportDialog::onWareshubConnectButtonClicked()
 
   try
   {
-    mp_WaresHubImportWorker = new openfluid::waresdev::WaresHubImportWorker(WaresHubUrl,
-                                                                            ui->UsernameLineEdit->text(),
-                                                                            ui->PasswordLineEdit->text(),
-                                                                            ui->SslNoVerifyCheckBox->isChecked());
+    bool SslNoVerify = openfluid::base::PreferencesManager::instance()->isWaresdevGitSslNoVerify();
```

Comment:
check with sslNoVerify enabled/disabled in preferences

Reply:
sslNoVerify enabled/disabled working with wareshub and fluidhub